bittensor/_cli/commands/misc.py

```diff
@@ -28,13 +28,6 @@ class HelpCommand:
     @staticmethod
     def run (cli):
         sys.argv = [sys.argv[0], '--help']
-        # # Run miner.
-        # if cli.config.model == 'core_server':
```

Comment:
Should remove btcli run then?

Reply:
Might as well gut the whole HelpCommand class, ya?

Reply:
> Might as well gut the whole HelpCommand class, ya?

Yes for sure